IPython/core/interactiveshell.py

```diff
@@ -1922,10 +1880,7 @@ def showindentationerror(self):
 
     def init_readline(self):
         """Moved to terminal subclass, here only to simplify the init logic."""
```

Comment:
This docstring should probably say 'deprecated' now. It's no longer in the terminal subclass, now that the terminal subclass uses PTK.